source/MaterialXContrib/MaterialXNode/CreateMaterialXNodeCmd.cpp

```diff
@@ -50,7 +50,7 @@ void registerFragment(const MaterialXData& materialData, const std::string& ogsX
     MString fragmentNameM;
 
     // Register fragments with the manager if needed
-    const std::string& fragmentString = materialData.getFragmentWrapper();
+    const std::string& fragmentString = materialData.getFragmentSource();
```

Comment:
I believe "source" is somewhat more intuitive than "wrapper" because a wrapper may mean e.g. a pointer to a "wrapper" object while "source" implies a blob with the source code.